### PR TITLE
COM-12616

### PR DESCRIPTION
**Pull request by mcesariniflu**
AAC:
- The ISO_IEC_14496-26_2010 name was changed by MPEG4_AAC-ADTS 

H264:
- The JVT-Professional_profiles_V1 name was changed by JVT-Professional_profiles
- The JVT-SVC_V1 name was changed by JVT-SVC

	modified:   README.md
	modified:   pyproject.toml
	modified:   scripts/gen_jvt.py
	renamed:    test_suites/aac/ISO_IEC_14496-26_2010.json -> test_suites/aac/MPEG4_AAC-ADTS.json
	renamed:    test_suites/h264/JVT-Professional_profiles_V1.json -> test_suites/h264/JVT-Professional_profiles.json
	renamed:    test_suites/h264/JVT-SVC_V1.json -> test_suites/h264/JVT-SVC.json

### CODE REVIEW

File: README.md
Comment (rsanchez87) [reply README.md:97]: I think we can include new test suites in this list... what do you think @mcesariniflu?

**Review by rsanchez87 — CHANGES_REQUESTED** [review]
A PR should be created in flu-plugins to change the names too... it's **very important**:

image

File: README.md
Comment (mcesariniflu) [reply README.md:97]: - Related to PR in flu-plugins:
Ok. I will create a PR for flu-plugins modifying the names also.

- Related to ISO_IEC_13818-4_2004
I saw it but I didn't change it because it was not mentioned in this task.

- Related to the README.md file
I updated only the name of the test suit mentioned in this task because there is another task, which is to update the README.md file itself. See: https://fluendo.atlassian.net/browse/COM-12613
That's the reason why I just modified the necessary things to meet this task.

Comment (rsanchez87) [reply README.md:97]: Ok, perfect, I hadn't seen the task you commented or I didn't remember, honestly I think the ones I have commented about the test suites that are changing (ISO_IEC_13818-4_2004 -> MPEG2_AAC-ADIF) are related to this task, but I understand.

Regarding the flu-plugins task let me know if you need me to review

Comment (mcesariniflu) [reply README.md:97]: Ok, I will update the ISO_IEC_13818-4_2004 to MPEG2_AAC-ADIF. It won't take too much effort.
Yes, I'll let you know so you can do the review.

**Review by rsanchez87 — APPROVED** [review]
LGTM
Wait at flu-plugins changes before merge [](https://github.com/fluendo/flu-plugins/pull/677)

**Pull request merged**
